include/audio/effects.h

```diff
@@ -0,0 +1,117 @@
+#ifndef AUDIO_EFFECTS_H
+#define AUDIO_EFFECTS_H
+
+#include "PR/ultratypes.h"
+#include "unk.h"
+
+struct Note;
+struct SequencePlayer;
+
+/* Multi-Point ADSR Envelope (Attack, Decay, Sustain, Release) */
+
+typedef enum AdsrStatus {
+    /* 0 */ ADSR_STATE_DISABLED,
+    /* 1 */ ADSR_STATE_INITIAL,
+    /* 2 */ ADSR_STATE_START_LOOP,
+    /* 3 */ ADSR_STATE_LOOP,
+    /* 4 */ ADSR_STATE_FADE,
+    /* 5 */ ADSR_STATE_HANG,
+    /* 6 */ ADSR_STATE_DECAY,
+    /* 7 */ ADSR_STATE_RELEASE,
+    /* 8 */ ADSR_STATE_SUSTAIN
+} AdsrStatus;
```

Comment:
Are these meant to be `state`s or `status`es? Also want to be careful with conflicting with the `AdsrState` struct (which is probably why the enum name is `Status`.

Reply:
Oh nice catch. Yeah I'll review and see which is appropriate where